### ISSUE TITLE
Charging Notifications properties...

### ISSUE THREAD

**Issue by ahzf** [hubject/oicp#61]
I wonder why the identification/authentication data is only optional within the charging notifications.
The most valueable use case for this is informing the end customer about the charging process, or?
Therefore it should be mandatory in my eyes.

Is the OperatorID within a charging notifications the same as a HubOperatorID within a charge detail record?

SessionStart is mandatory for charge detail record since OICP v2.3, but (still) optional within charging notifications. Perhaps it would be useful to make it also mandatory in charging notifications.

For the German Eichrecht we have some energy meters which will always output a signed meter value if the start value was a signed meter value. When we use signed meter values we also have to include ALL signed meter values into the later CDR, otherwise the CDR is not legally valid. Therefore it would be usefull if we could also send signed meter values within charging notifications... we have all the data anyway.

**Comment by will-afs** [comment]
Yes the eRoamingChargingNotificationsStart is inconsistent between the [Swagger docs](https://hubject.github.io/oicp-cpo-2.3-api-doc/#tag/eRoamingChargingNotifications/operation/eRoamingChargingNotifications_V1.1) and the implementation proposed in this repo:
Identification is a required field in the first and optional in the second.
